src/scenario/ScenarioRepository.cpp

```diff
@@ -24,6 +24,7 @@
 #include "../core/Path.hpp"
 #include "../core/String.hpp"
 #include "../PlatformEnvironment.h"
+#include "../core/Util.hpp"
```

Comment:
This should go above `PlatformEnvironment` if you want to keep the headers sorted.